ado_asana_sync/sync/sync.py

```diff
@@ -158,6 +159,27 @@
         return True
 
 
+def safe_get(obj, *attrs_keys):
+    """
+    Safely retrieves nested attributes from an object.
+
+    Args:
+        obj: The object to retrieve attributes from.
+        *attrs_keys: Variable number of attribute keys.
+
+    Returns:
+        The value of the nested attribute if found, else None.
+    """
+    for attr_key in attrs_keys:
+        if isinstance(obj, dict):
+            obj = obj.get(attr_key)
+        else:
+            obj = getattr(obj, attr_key, None)
+        if obj is None:
+            return None
+    return obj
```

Comment:
**Suggestion:** The `safe_get` function is used to access a nested attribute in a dictionary or an object. However, the function will fail if one of the intermediate keys does not exist. It would be better to add a check to ensure that the intermediate keys exist before trying to access them.
```suggestion
def safe_get(obj, *attrs_keys):
    for attr_key in attrs_keys:
        if isinstance(obj, dict) and attr_key in obj:
            obj = obj.get(attr_key)
        elif hasattr(obj, attr_key):
            obj = getattr(obj, attr_key)
        else:
            return None
    return obj
```